backend/pubsub.js

```diff
@@ -231,14 +234,16 @@ const defaultSocketEventHandlers = {
     }
     // The socket can be marked as active since it just received a message.
     socket.activeSinceLastPing = true
-    const handler = server.messageHandlers[msg.type]
+    const defaultHandler = defaultMessageHandlers[msg.type]
+    const customHandler = server.customMessageHandlers[msg.type]
 
-    if (handler) {
+    if (defaultHandler || customHandler) {
       try {
-        handler.call(socket, msg)
+        defaultHandler?.call(socket, msg)
+        customHandler?.call(socket, msg)
```

Comment:
Because I needed to define custom handlers (e.g., for the `close` event) and it didn't make sense to have to rewrite all of the logic we already have. In fact, I can't think of many situations where one would ever want to entirely override the default handler (and if the need arises, it could be an explicit opt-in)